icu4c/source/test/fuzzer/fuzzer_driver.cpp

```diff
@@ -0,0 +1,20 @@
+// © 2019 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "cmemory.h"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size);
```

Comment:
why is this called `LLVMFuzzerTestOneInput` ? Is this only meant to be run in an llvm context?

Reply:
I assume this is here so these can be verified like a test.  Does the 'actual' fuzzer mechanism use this file, the individual .o's, or the individual binaries?

Reply:
Yes. fuzzer_driver performs a basic smoke test of the fuzzer targets.
The fuzzer requires LLVMFuzzerTestOneInput().

Reply:
I believe the actual fuzzer mechanism links with the .o, and expects to find LLVMFuzzerTestOneInput () therein.

Reply:
The actual fuzzer mechanism does not use my fuzzer_driver.cpp. Fuzzer compiles e.g. break_iterator_fuzzer.cpp, links break_iterator_fuzzer.o together with the fuzzer code into one executable and then calls LLVMFuzzerTestOneInput(...) 
fuzzer_driver.cpp is only used on side of ICU for smoke testing the fuzzer target.